### ISSUE TITLE
Loading ksml.lua

### ISSUE THREAD

**Issue by bauen1** [BTCTaras/kristscape#12]
How are you gonna go about this, because `os.loadAPI` also places the api in `_G` and everything is local.
Oh and should i try to convert tabs to classes?

**Comment by BTCTaras** [comment]
Most likely with `loadstring` but we'll see. The file would probably be signed before a release, so there'd be an error if it is tampered with.

Tabs are working nicely already. I don't think we need to change them, but if you do, you're free to PR

**Comment by bauen1** [comment]
I would go like this

```
local function class ()
    local c = {}
    c.__index = c

    local mt = {}
    mt.__call = function (cls, ...)
        local obj = {}
        setmetatable (obj, c)

        if cls.init then
            cls.init (obj, ...)
        end

        return obj
    end

    setmetatable (c, mt)
    return c
end

local tabClass = class ()

function tabClass:init (i)
    self.name = "my_name"
    self.value = i
end

function tabClass:add_one (v)
    self.value = self.value + (v or 1)
end

function tabClass:__tostring ()
    return self.name .. "@" .. self.value
end

local instance = tabClass (-2)
print (instance)
print (instance.value)
instance:add_one ()
print (instance.value)
instance:add_one (9)
print (instance.value)
print (instance)

os.sleep (10)
```

It allowes to acces variables by `instance.var_name` and method calling by `instance:method (args)`
If thats ok with you

**Comment by bauen1** [comment]
I made a commit to my fork containing this change, and its working (mostly) but still needs some tweaks to make it "good" code again :smile:

**Comment by BTCTaras** [comment]
Just merged #13
